src/l10n/fr.js

```diff
@@ -0,0 +1,357 @@
+(function() {
+    let translations = {
+            'Passwords'                           : 'Mots de passe',
+            'All'                                 : 'Tous',
+            'Folders'                             : 'Dossiers',
+            'Recent'                              : 'Récents',
+            'Favorites'                           : 'Favoris',
+            'Shared'                              : 'Partagés',
+            'Tags'                                : 'Mots clés',
+            'Security'                            : 'Sécurité',
+            'Trash'                               : 'Corbeille',
+            'Details'                             : 'Détails',
+            'Edit'                                : 'Modifier',
+            'Rename'                              : 'Renommer',
+            'Delete'                              : 'Effacer',
+            'Restore'                             : 'Restaurer',
+            'New Password'                        : 'Nouveau mots de passe',
+            'New Folder'                          : 'Nouveau dossier',
+            'New Tag'                             : 'Nouveau mot clé',
+            'Secure'                              : 'Sécurisé',
+            'Weak'                                : 'Faible',
+            'Weak (Duplicate)'                    : 'Faible (Dupliqué)',
+            'Weak (Outdated)'                     : 'Faible (Obsolète)',
+            'Breached'                            : 'Compromis',
+            'Notes'                               : 'Notes',
+            'Share'                               : 'Partager',
+            'Revisions'                           : 'Révisions',
+            'QR Code'                             : 'QR Code',
+            'Password'                            : 'Mot de passe',
+            'Username'                            : 'Nom d’utilisateur',
+            'Website'                             : 'Site internet',
+            'Add Tags...'                         : 'Ajouter des mots clés…',
```

Comment:
I do not know if this was meant like that or if it's a mistake, but you have a special character there for the 3 dots at the end.

Reply:
it's not a mistake it's the correct character in french, i prefer use the unique chara >…< in place to the 3 dots >...<

Reply:
Okay 👍